tests/custom_hypothesis_support.py

```diff
@@ -276,6 +281,60 @@ def simple_homogeneous_set_node(draw, **kwargs):
     t = draw(primitive_types)
     return set_node(const_node(t()), **kwargs).example()
 
+
+@hs.composite
+def name_node(draw, name=None):
+    if not name:
+        node = astroid.Name(draw(valid_identifier()))
+    else:
+        node = astroid.Name(draw(name))
+    return node
+
+
+@hs.composite
+def arguments_node(draw, annotated=False):
+    n = draw(hs.integers(min_value=1, max_value=5))
+    args = draw(hs.lists(name_node(None), min_size=n, max_size=n))
+    if annotated:
+        # TODO: annotations have to be Name nodes BOI
```

Comment:
@RyanDJLee in general you should search for TODOs and remove them before committing your work.